fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java

```diff
@@ -723,6 +726,7 @@ public void testSearchIncludeIteratePrimaryType() {
         WebTarget target = getWebTarget();
         Response response =
                 target.path("Patient")
+                .queryParam("_total", "none")
```

Comment:
could add an assertNull(bundle.getTotal()) in this testcase

Reply:
Added.